ddtrace/opentelemetry/_context.py

```diff
@@ -32,8 +35,9 @@ def attach(self, otel_context):
             if isinstance(otel_span, Span):
                 self._ddcontext_provider.activate(otel_span._ddspan)
             elif isinstance(otel_span, OtelSpan):
-                trace_id, span_id, *_ = otel_span.get_span_context()
```

Comment:
Here sampling configurations in tracestate and traceflags were consistently ignored. This is no longer the case.